smdebug/core/hook.py

```diff
@@ -390,16 +390,14 @@ def _get_collections_with_tensor(self, tensor_name) -> Set["Collection"]:
     def _get_default_collections(self):
         pass
 
-    def has_default_hook_configuration(self):
+    def has_default_hook_configuration(self, default_saved_collections=DEFAULT_SAVED_COLLECTIONS):
```

Comment:
Irrelevant change. Refactoring so that the default collections are passed in as an argument from KerasHook and XGBoost Hook instead of the implementation being completely duplicated.